tests/lib/test_traffic.py

```diff
@@ -55,6 +65,40 @@ def test_0030_expr_comparator(self):
         # test comparator expression
         correct_values = [(u.id, b > 0 if b is not None else None)
                           for u, b in self.correct_balance.items()]
-        res = session.session.query(User.id, traffic_balance_expr()>0).all()
+        res = session.session.query(User.id, traffic_balance_expr() > 0).all()
 
         self.assertEqual(set(correct_values), set(res))
+
+
+class Test_020_TrafficLimitExceeded(FixtureDataTestBase):
```

Comment:
:+1: for unprompted testing.
Remember to validate your test by _removing parts of the logic_ (e.g., removing a `has_property` filter) and confirming that the test actually fails! Else it's just a meaningless green light.